examples/src/test/scala/org/virtuslab/ideprobe/ModuleTest.scala

```diff
@@ -0,0 +1,53 @@
+package org.virtuslab.ideprobe
+
+import java.nio.file.Files
+
+import org.junit.Assert
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+import org.virtuslab.ideprobe.protocol.{BuildScope, JUnitRunConfiguration, ModuleRef, Setting}
+import org.virtuslab.intellij.scala.SbtTestSuite
+import org.virtuslab.intellij.scala.protocol.SbtProjectSettingsChangeRequest
+
+class ModuleTest extends SbtTestSuite {
+  @ParameterizedTest
+  @ValueSource(
+    strings = Array("projects/shapeless/ideprobe.conf", "projects/cats/ideprobe.conf", "projects/dokka/ideprobe.conf")
+  )
+  def verifyModulesPresent(configName: String): Unit = fixtureFromConfig(configName).run { intelliJ =>
+    deleteIdeaSettings(intelliJ)
+    intelliJ.probe.openProject(intelliJ.workspace)
+    val project = intelliJ.probe.projectModel()
+    val projectModules = project.modules.map(_.name)
+    val modulesFromConfig = intelliJ.config[Seq[String]]("modules.verify")
+
+    val missingModules = modulesFromConfig.diff(projectModules)
+    Assert.assertTrue(s"Modules $missingModules are missing", missingModules.isEmpty)
+  }
+
+  @ParameterizedTest
+  @ValueSource(
+    strings = Array("projects/shapeless/ideprobe.conf", "projects/cats/ideprobe.conf", "projects/dokka/ideprobe.conf")
```

Comment:
if there is nothing in shapeless or cats or dokka directory it is porbably simpler to just have `projects/shapeless.conf` etc.

Reply:
done